### PR TITLE
Add aggregator

### PR DESCRIPTION
**Pull request by deifyed**
This PR: 
- Adds a setup guide for a harvester user on AWS
- Adds a region setting for the AWS harvester
- Adds secret handling for poctools publishing. `make publish` will now automatically publish to PyPi. (Remember to `gopass sync`)
- Employs a new folder structure to handle aggregations and datasets. The folder structure is documented in /docs/poc_pipeline_flow.png, but looks something like /data/{dataservice,dataset}/{10_raw,20_aggregations,30_result}/
- Adds an aggregator which based on a visibility.json, name.json, specificationURL.json and description.json enriches the apis harvested in /data/dataservice/10_raw and exports the result to /data/dataservice/30_result/{protected,public}.json

### CODE REVIEW

File: tools/aggregator/aggregator.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import json
```
Comment (yngvark) [inline tools/aggregator/aggregator.py:1]: Kan vi lage en aggregator/README.md som sier hva aggregatoren gjør?

**Review by yngvark — CHANGES_REQUESTED** [review]

File: tools/aggregator/aggregator.py (hunk 1 above)
Comment (deifyed) [reply tools/aggregator/aggregator.py:1]: /giphy erna vil ikke

Comment (deifyed) [reply tools/aggregator/aggregator.py:1]: (joda. veldig godt poeng)

**Review by yngvark — APPROVED** [review]
Muito bom

**Pull request merged**